tests/bytes_test.lua

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
we don't need test case, but this is a good example to show user how to use transform.
maybe change this file name to `transform_example.lua` and put this file together with `transform.lua`
CC @sauvainr

Reply:
yes, for clarity the functions are all in parser_factory and some example in _vendor/transform.lua_